src/atproto.rs

```diff
@@ -539,6 +542,33 @@ impl Client {
             Err(StreamError::NoCursor)
         }
     }
+
+    /// com.atproto.repo.putRecord
+    pub async fn repo_put_record<T: Serialize>(
```

Comment:
This is long enough that maybe we should just take the `PutRecord` as the arg?

Reply:
Oky